client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -636,34 +642,35 @@ const JetpackConnectAuthorizeForm = React.createClass( {
 
 export default connect(
 	state => {
-		const queryObjectSite = state.jetpackConnect.jetpackConnectAuthorize &&
-			state.jetpackConnect.jetpackConnectAuthorize.queryObject &&
-			state.jetpackConnect.jetpackConnectAuthorize.queryObject.site;
-
-		const site = queryObjectSite
-			? getSiteByUrl( state, state.jetpackConnect.jetpackConnectAuthorize.queryObject.site )
+		const authorizeSite = getAuthorizeSite( state );
```

Comment:
Yeah, that's a good point, these 3 selectors (`getAuthorize*`) can be a little more informative.